docs/concepts/flatcar-container-linux.md

```diff
@@ -0,0 +1,83 @@
+# Operating system
+
+## Flatcar Container Linux
+
+Lokomotive installs [Flatcar Container Linux](https://www.flatcar-linux.org/) as the underlying operating system.
+
+Flatcar Container Linux is an open source immutable Linux distribution for containers. It is a
+friendly fork of CoreOS Container Linux and as such, compatible with it.
+
+### Why Flatcar Container Linux ?
+
+* Minimal distribution required for containers.
+  * Reduced dependencies.
+  * Reduced attack surface area.
+* Immutable file system.
+  * Operational simplicity.
+  * Removes entire category of security threats. (e.g. runc vulnerability CVE-2019-5736)
+    https://kinvolk.io/blog/2019/02/runc-breakout-vulnerability-mitigated-on-flatcar-linux/
+* Automated, streamlined and atomic updates.
+  * Easily apply all latest security patches.
+  * Rollback partition.
+* Declarative and immutable configuration.
+* Optimization for containerized applications.
+
+### Properties
+
+Flatcar Container Linux specific properties.
+
+### Host properties
+
+| Property               | Flatcar Container Linux |
+|------------------------|-------------------------|
+| host spec (bare-metal) | Container Linux Config  |
```

Comment:
What is "host spec"? I don't think this is a term commonly used and I'm not sure it's useful in here. I'd remove these two.